Fst2List.cpp

```diff
@@ -1243,6 +1335,324 @@ class CFstApp {
           pathStack[i].stateNo, pathStack[i].tag);
   }
 
+  static void update_last_position(struct locate_parameters* p, int pos) {
+    if (pos > p->last_tested_position) {
+      p->last_tested_position = pos;
+    }
+  }
+
+  /**
+    * check if the given lexical mask is already processed
+    * output is its output (may be NULL)
```

Comment:
`output is its output` could you rephrase?

Reply:
done